lib/engines/content_block_manager/app/services/content_block_manager/preview_content.rb

```diff
@@ -0,0 +1,67 @@
+require "net/http"
+require "json"
+require "uri"
+
+module ContentBlockManager
+  class PreviewContent < Data.define(:title, :html)
```

Comment:
This might sit better in models now

Reply:
ah ok, is that the case even though it's making a call out to other APIs? Always though Models was more for the actual db models or like flat data

Reply:
actually making me think what we need is a flat Preview Content and then a Get Preview Content service, like we have for Host Content Item?

Reply:
That would probably be more consistent actually

Reply:
> making me think what we need is a flat Preview Content and then a Get Preview Content service, like we have for Host Content Item?

I've gone with this approach for consistency and keeping things that call outside of the app in service folder.